Automattic/wp-calypso
# Marketplace: Adds Education Footer

#### Changes proposed in this Pull Request

* Adds education footer

#### Screenshots
| Mobile | Tablet | Desktop |
|--------|-------|----------|
|<img width="322" alt="Screen Shot 2022-03-08 at 7 09 13 PM" src="https://user-images.githubusercontent.com/1035546/157347308-ec85e963-3872-4290-8b8d-aa65952e12e0.png">|<img width="771" alt="Screen Shot 2022-03-08 at 7 08 49 PM" src="https://user-images.githubusercontent.com/1035546/157347295-593f5522-94ec-4ff5-944e-7f39d3aa5b0e.png">|<img width="1512" alt="Screen Shot 2022-03-07 at 11 32 25 PM" src="https://user-images.githubusercontent.com/1035546/157166627-e7cafe25-81f1-4a4b-bf94-d73377050e1b.png">|




#### Testing instructions



* Navigate to `http://calypso.localhost:3000/plugins/:siteId`
* Scroll down to the bottom
* Footer should appear
* Clicking on cards should link to the correct wordpress.com go article



Related to #61491
